svf/include/SVFIR/SVFIR.h

```diff
@@ -741,14 +725,18 @@ class SVFIR : public IRGraph
     inline NodeID addBlackholeObjNode()
     {
         return addObjNode(
-                   nullptr, new DummyObjVar(getBlackHoleNode(), getBlackHoleObj()),
-                   getBlackHoleNode());
+                   nullptr, new DummyObjVar(getBlackHoleNode(),
```

Comment:
Reduce the number of arguments. The first and last arguments are the same.

pass ObjtypeInfo directly.